src/routes/v1/doc/operation/terminal.go

```diff
@@ -0,0 +1,171 @@
+				Example:  defaultExample,
+			},
+			{
+				Name:     namespaceNameKey,
+				In:       pathKey,
+				Required: true,
+				Schema:   huma.SchemaFromType(registry, reflect.TypeOf(types.StartTerminalUri{}.NamespaceName)),
+				Example:  defaultExample,
+			},
+			{
+				Name:     podNameKey,
+				In:       pathKey,
+				Required: true,
+				Schema:   huma.SchemaFromType(registry, reflect.TypeOf(types.StartTerminalUri{}.PodName)),
+				Example:  defaultExample,
+			},
+			{
+				Name:     containerNameKey,
+				In:       pathKey,
+				Required: true,
+				Schema:   huma.SchemaFromType(registry, reflect.TypeOf(types.StartTerminalUri{}.ContainerName)),
+				Example:  defaultExample,
+			},
+		},
+		RequestBody: &huma.RequestBody{
+			Content: map[string]*huma.MediaType{
+				applicationJSONKey: {
+					Schema: huma.SchemaFromType(registry, reflect.TypeOf(types.StartTerminalBody{})),
+				},
+			},
+		},
+		Security: []map[string][]string{
+			{bearerKey: {}},
+		},
+		Responses: map[string]*huma.Response{
+			strconv.Itoa(http.StatusOK): {
+				Content: map[string]*huma.MediaType{
+					applicationJSONKey: {
+						Schema: huma.SchemaFromType(registry, reflect.TypeOf(types.StartTerminalResponse{})),
+					},
+				},
+			},
+			strconv.Itoa(http.StatusBadRequest): {
+				Content: map[string]*huma.MediaType{
+					applicationJSONKey: {
+						Schema: huma.SchemaFromType(registry, reflect.TypeOf(types.ErrorResponse{})),
+					},
+				},
+			},
+			strconv.Itoa(http.StatusInternalServerError): {
+				Content: map[string]*huma.MediaType{
+					applicationJSONKey: {
+						Schema: huma.SchemaFromType(registry, reflect.TypeOf(types.ErrorResponse{})),
+					},
+				},
+			},
+		},
+	}
+	api.OpenAPI().AddOperation(operation)
+}
+
+// AddServeTerminal adds the StartTerminal route to the OpenAPI scheme.
+func AddServeTerminal(api huma.API, registry huma.Registry) {
+	operation := &huma.Operation{
+		OperationID: "serve-terminal",
+		Method:      http.MethodGet,
+		Tags:        []string{terminalTag},
+		Path:        fmt.Sprintf("/ws/%s", terminalKey),
+		Summary:     "Serves terminal for a pod in a namespace",
+		Description: "Serves an interactive terminal for a pod using websockets",
+		Parameters: []*huma.Param{
+			{
+				Name:     connectionHeaderKey,
+				In:       headerKey,
+				Required: true,
+				Schema:   huma.SchemaFromType(registry, reflect.TypeOf(huma.TypeString)),
+				Example:  upgradeHeaderKey,
+			},
+			{
+				Name:     upgradeHeaderKey,
+				In:       headerKey,
+				Required: true,
+				Schema:   huma.SchemaFromType(registry, reflect.TypeOf(huma.TypeString)),
+				Example:  webSocketValue,
+			},
+			{
+				Name:     secWebSocketVersionHeaderKey,
+				In:       headerKey,
+				Required: true,
+				Schema:   huma.SchemaFromType(registry, reflect.TypeOf(huma.TypeString)),
+				Example:  secWebSocketVersionValue,
+			},
+		},
+		RequestBody: &huma.RequestBody{
+			Content: map[string]*huma.MediaType{
+				applicationJSONKey: {
```

Comment:
Are we sure this is application/json?
Does this work?

Reply:
So none of the websockets stuff (logs/terminal) actually work from the `/docs` endpoint, this serves more as an example to use in clients that can actually do websockets (i.e. Postman)